bears/uploadTool.py

```diff
@@ -0,0 +1,81 @@
+import glob
+import os
+import re
+import shutil
+import sys
+
+sys.path.insert(0, '..')
+
+from bears.Constants import classifiers
+from bears.Constants import encoding
+from bears.Constants import imports
+from coalib.collecting.Importers import iimport_objects
+
+
+def main():
+    # create dir and set the path to the working directory
+    if not os.path.exists('upload'):
+        os.mkdir('upload')
+
+    bears_full_filenames = glob.glob('**/*Bear.py')
+
+    for bear_file_name in bears_full_filenames:
+        file_name = bear_file_name
+        bear_name = re.match(r'.+/(.*)\.py', bear_file_name).group(1)
+        bear_object = list(iimport_objects(file_name, attributes='kind',
+                                           local=True))
+        if not os.path.exists('upload/' + bear_name):
+            os.mkdir(os.path.join('upload', bear_name))
+        if not os.path.exists(os.path.join('upload', bear_name, bear_name)):
+            os.mkdir(os.path.join('upload', bear_name, bear_name))
+        if not os.path.exists(os.path.join('upload', bear_name, bear_name,
+                                           '__init__.py')):
+            init = open(os.path.join('upload', bear_name, bear_name,
+                                     '__init__.py'), 'w')
+            init.write(' ')
+            init.close()
```

Comment:
Simplify to `open(os.path.join('upload', bear_name, bear_name, '__init__.py'), 'w').close()`